src/model/configuration.go

```diff
@@ -282,6 +306,9 @@ func (m *Configuration) saveConfiguration(tx *sql.Tx) error {
 		admin_pwd = ?,
 		pickup_mode = ?,
 		lock_entry_list = ?,
+		race_pit_window_start = ?,
```

Comment:
Move this after udp_plugin_address to match database table order and adjust parameters.